ngx-fudis/projects/documentation/development/tooling/TestingLocalChanges.mdx

```diff
@@ -0,0 +1,37 @@
+import { Meta } from "@storybook/blocks";
+
+<Meta title="Documentation/Development/Tooling/Testing Local Changes in Application" />
+
+# Testing Local Changes in Application
+
+Sometimes you need to test your local changes straight away in the application. The Fudis package may be either a symlinked workspace or a packaged tarball.
+Instructions cover both NPM and Yarn package managers.
+
+## Testing with Symlink
+
+### NPM
+
+1. Navigate to **ngx-fudis** directory.
+2. Build the library: `npm run build:ngx-fudis`. New changes require rebuilding the library again.
+3. Go to the **ngx-fudis/dist** directory and run `npm link` that creates a local symlink to the Fudis package.
+4. Then navigate to your project where you want to link the Fudis package. The directory that contains the node_modules folder containing the project dependencies is the correct place to execute linking. In this project directory run `npm link @funidata/ngx-fudis`
+5. When you want to unlink the package from your project, run the command `npm unlink @funidata/ngx-fudis`. This is done in the same directory in your consuming project, where you linked the package.
```

Comment:
I see you have deleted a part (which was originally only in Core) but I think it is good practice to be here as a 6th step.

> 6. If you want to remove the global symlink, go back to the **ngx-fudis/dist** folder and run `npm unlink`.

Otherwise it will hunt you for the second time when trying to use npm link.

Reply:
Maybe this would need `npm unlink -g` (reference https://stackoverflow.com/questions/69906798/unable-to-remove-symlinks-after-running-npm-link)